app/models/org.rb

```diff
@@ -88,20 +93,14 @@ def get_locale
   #
   # @return [String]
   def type
-    if self.institution?
-      return "Institution"
-    elsif self.funder?
-      return "Funder"
-    elsif self.organisation?
-      return "Organisation"
-    elsif self.research_institute?
-      return "Research Institute"
-    elsif self.project?
-      return "Project"
-    elsif self.school?
-      return "School"
-    end
-      return "None"
+    ret = []
```

Comment:
@briri, this method will conflict with the ActiveRecord type method used for Single Table Inheritance and we might have side effects. Can you rename it to something like "org_type_to_s"?

Reply:
yes good catch